skid_steer_drive_controller:
  type: pandora_motor/SkidSteerDriveController
  wheel_separation: 0.36
  wheel_radius: 0.0975
  max_angular_velocity: 6135
  gearbox_ratio: 113
  joints:
    - movement_left_front_joint
    - movement_right_front_joint
    - movement_left_rear_joint
    - movement_right_rear_joint
